src/components/modals/AutoLogoutModal.tsx

```diff
@@ -1,108 +1,93 @@
 import { Picker } from '@react-native-picker/picker'
 import * as React from 'react'
-import { Appearance, Platform, Text, View } from 'react-native'
+import { View } from 'react-native'
 import { AirshipBridge } from 'react-native-airship'
-import IonIcon from 'react-native-vector-icons/Ionicons'
 
+import { useHandler } from '../../hooks/useHandler'
 import { lstrings } from '../../locales/strings'
-import { dayText } from '../../styles/common/textStyles'
-import { THEME } from '../../theme/variables/airbitz'
 import { DisplayTime, displayToSeconds, secondsToDisplay } from '../../util/displayTime'
-import { AirshipModal } from '../legacy/AirshipModal'
-import { PrimaryButton } from '../legacy/Buttons/PrimaryButton.ui'
-import { SecondaryButton } from '../legacy/Buttons/SecondaryButton.ui'
-import { ContentArea } from '../legacy/ContentArea'
-import { IconCircle } from '../legacy/IconCircle'
-import { LadderLayout } from '../legacy/LadderLayout'
-import { ThemeProps, withTheme } from '../services/ThemeContext'
+import { ButtonsContainer } from '../buttons/ButtonsContainer'
+import { cacheStyles, Theme, useTheme } from '../services/ThemeContext'
+import { ModalTitle } from '../themed/ModalParts'
+import { ThemedModal } from '../themed/ThemedModal'
 
-interface OwnProps {
+interface Props {
   bridge: AirshipBridge<number | undefined>
   autoLogoutTimeInSeconds: number
 }
 
-type State = DisplayTime
+export const AutoLogoutModal = (props: Props) => {
+  const { autoLogoutTimeInSeconds, bridge } = props
+  const theme = useTheme()
+  const styles = getStyles(theme)
+  const textColor = theme.pickerTextDark
```

Comment:
Would it make sense to delete `pickerTextDark` and `pickerTextLight` from the theme, and then just use normal `theme.primaryText` for this? Or, at the very least, just merge the two into a single `pickerText` entry?